ADA/databricks_notebooks.qmd

```diff
@@ -11,15 +11,15 @@ title: "Databricks Notebooks"
 Notebooks are a special kind of script that Databricks supports. They consist of code blocks and markdown blocks which can contain formatted text, links and images. Due to the ability to combine markdown with code they are very well suited to creating and documenting data pipelines, such as creating a core dataset that underpins your other products. They are particularly powerful when parameterised and used in conjuction with [Workflows](ADA/Databricks_workflows.qmd).
 
 You can create a notebook in your workspace, either in a folder or a repository.\
```

Comment:
Why are we using the \ for a new line, should we just have a blank line between them instead?

![image](https://github.com/user-attachments/assets/e73d800d-f382-4ab9-945e-b8ad56613359)

Reply:
I suspect this is something that RStudio does when you edit in visual mode. In this case I think there's an argument for no line break as it could all be part of the same paragraph?

Reply:
Yeah I'd make this all one paragraph

Reply:
Done 😄